Assets/MixedRealityToolkit.Examples/Experimental/Audio/Scripts/MicrophoneAmplitudeDemo.cs

```diff
@@ -0,0 +1,208 @@
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Experimental.Audio;
+using Microsoft.MixedReality.Toolkit.SpatialAwareness;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Examples.Experimental
+{
+    /// <summary>
+    /// Demonstration class using <see cref="WindowsMicrophoneStream"/> to select the voice microphone and
+    /// adjust the spatial awareness mesh based on the amplitude of the user's voice.
+    /// </summary>
+    [RequireComponent(typeof(AudioSource))]
+    public class MicrophoneAmplitudeDemo : MonoBehaviour
+    {
+        [SerializeField]
+        [Tooltip("Factor by which to boost the microphone amplitude when changing the mesh display.")]
+        [Range(0, 50)]
+        private int amplitudeBoostFactor = 10;
+
+        private IMixedRealitySpatialAwarenessSystem spatialAwarenessSystem = null;
+
+        /// <summary>
+        /// Instance of the spatial awareness system.
+        /// </summary>
+        private IMixedRealitySpatialAwarenessSystem SpatialAwarenessSystem
+        {
+            get
+            {
+                if (spatialAwarenessSystem == null)
+                {
+                    MixedRealityServiceRegistry.TryGetService<IMixedRealitySpatialAwarenessSystem>(out spatialAwarenessSystem);
+                }
+                return spatialAwarenessSystem;
+            }
+        }
+
+        private IMixedRealitySpatialAwarenessMeshObserver spatialMeshObserver = null;
+
+        /// <summary>
+        /// Instance of a spatial awareness mesh observer. Used to acquire the visible mesh material.
+        /// </summary>
+        private IMixedRealitySpatialAwarenessMeshObserver SpatialMeshObserver
+        {
+            get
+            {
+                if (spatialMeshObserver == null)
+                {
+                    spatialMeshObserver = (SpatialAwarenessSystem as IMixedRealityDataProviderAccess)?.GetDataProvider<IMixedRealitySpatialAwarenessMeshObserver>();
+                }
+                return spatialMeshObserver;
+            }
+        }
+
+        private Material visibleMaterial = null;
+
+        /// <summary>
+        /// The visuble mesh material in use by the spatial mesh observer.
+        /// </summary>
+        private Material VisibleMaterial
+        {
+            get
+            {
+                if (visibleMaterial == null)
+                {
+                    visibleMaterial = SpatialMeshObserver?.VisibleMaterial;
+                }
+                return visibleMaterial;
+            }
+        }
+
+        /// <summary>
+        /// Class providing microphone stream management support on Microsoft Windows based devices.
+        /// </summary>
+        private WindowsMicrophoneStream micStream = null;
+
+        /// <summary>
+        /// The average amplitude of the sound captured during the most recent microphone update.
```

Comment:
> The average amplitude of the sound captured during the most recent microphone update. [](start = 11, length = 86)

Is there a numerical scale here (i.e. 0-100? 0-1?) 

Is it decibels?

Reply:
0-1.